SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -236,27 +236,29 @@ private Task ProcessPendingErrorsAsync()
                         }
                         continue;
                     }
-                    var report = BuildErrorReport(log);
-                    if (report == null)
+
+                    // The error report cannot be built if any of those fields are null.
+                    if (log.Device == null || log.AppLaunchTimestamp == null || log.Timestamp == null)
```

Comment:
Would calling Validate() on the log make sense here?

Reply:
It is a generated data model I guess.

Reply:
[Here](https://github.com/microsoft/appcenter-sdk-dotnet/blob/develop/scripts/generated/Models/Log.cs)
In the generated model, `Device` is the only one nonnull.

Reply:
I also had a same doubt but I believed my assumption is true. 😄